modules/test/playwright/tests/dynamic-data-mapping-form-web/formRules.spec.ts

```diff
@@ -56,6 +56,174 @@ test.beforeEach(({page}) => {
 	page.setViewportSize({height: 1080, width: 1920});
 });
 
+test('Assert multiple selection change with predefined value', async ({
```

Comment:
Hi @rtienpal, could you mention the form rule in the test name? I believe this is an important piece of information that can be included here.